docs/integration/assets_categories/device/aws_ec2_device.md

```diff
@@ -0,0 +1,133 @@
+## Overview
+
+Amazon Elastic Compute Cloud (EC2) is a web service that provides secure, resizable compute capacity in the cloud. It is designed to make web-scale cloud computing easier for developers by providing a simple web service interface that allows you to obtain and configure capacity with minimal friction.
+
+- **Vendor**: Amazon
+- **Product**: AWS EC2
+- **Supported environment**: Cloud
+
+## Configure
+
+### How to create an IAM account with proper permissions
+
+To connect AWS EC2 to Sekoia.io, you need to create an IAM user with the necessary permissions to access EC2 instances. Follow these steps:
+
+1. Sign in to the AWS Management Console and navigate to the IAM service.
+
+    ![AWS Console IAM service selection](/assets/operation_center/asset_connectors/device/aws/aws_create_user_1.png)
+
+2. Click **Users** in the left navigation pane, then click **Create user**.
+
+    ![IAM Users page with Create user button highlighted](/assets/operation_center/asset_connectors/device/aws/aws_create_user_2.png)
+
+3. Enter a username for the IAM user (e.g., `sekoia-ec2-reader`) and click **Next**.
+
+    ![Create user form with username field](/assets/operation_center/asset_connectors/device/aws/aws_create_user_3.png)
+
+4. In the **Set permissions** step, select **Attach policies directly** and search for the following policies:
+   - `AmazonEC2ReadOnlyAccess` (for basic EC2 read access)
+   - `AmazonEC2FullAccess` (if you need full EC2 access)
```

Comment:
I think that the tabulation is not enough.
In the preview, the three permissions appeared as steps